docs/cni.md

```diff
@@ -116,7 +116,16 @@ For example:
 ## Custom networks
 
 You can also customize your CNI network by providing configuration files.
-For example you have one configuration file(`/etc/cni/net.d/10-mynet.conf`)
+
+When rootful, the expected root location is `/etc/cni/net.d`.
+For rootless, the expected root location is `~/.config/cni/net.d/`
```

Comment:
Could you also update https://github.com/containerd/nerdctl/blob/main/docs/dir.md#cni

Reply:
Done